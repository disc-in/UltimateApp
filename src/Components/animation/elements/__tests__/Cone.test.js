import React from 'react';
import { render } from '@testing-library/react-native';

import Cone from '../Cone';

describe('<Cone />', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<Cone baseWidth={10} number={1} />);
    expect(toJSON()).toMatchSnapshot();
  });
});
