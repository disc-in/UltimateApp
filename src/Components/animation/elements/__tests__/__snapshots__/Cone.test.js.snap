// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cone /> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "transparent",
        "borderBottomColor": "#feae19",
        "borderColor": "transparent",
        "borderStyle": "solid",
        "justifyContent": "center",
        "textAlign": "center",
        "textAlignVertical": "center",
      },
      Object {
        "borderBottomWidth": 5,
        "borderLeftWidth": 2.5,
        "borderRightWidth": 2.5,
      },
    ]
  }
>
  <Text
    style={
      Object {
        "height": 0,
        "width": 0,
      }
    }
  >
    1
  </Text>
</View>
`;
